.github/workflows/cast_vlaura_votes.yaml

```diff
@@ -0,0 +1,44 @@
+name: Post vlAURA snapshot votes to voter multisig and send to the vote relayer
+
+on:
+  workflow_dispatch:
+    inputs:
+      manual:
+        description: 'Use manual mode (true/false). Manual votes are placed in `tools/python/aura_snapshot_voting/manual_voting/{pool_type}.csv`'
+        required: true
+
+
+jobs:
+  generate_fees_report:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Setup Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+
+    - name: vlAURA Voting
+      env:
+        ETHNODEURL: ${{ secrets.ETHNODEURL }}
+        PRIVATE_KEY: ${{ secrets.PRIVATE_KEY }}
+        DUNE_API_KEY: ${{ secrets.DUNE_API_KEY }}
+      run: |
+        pwd
+        cd tools/python
+        pip3 install -r requirements.txt
+        cd aura_snapshot_voting
+        python3 vote.py ${{ github.event.inputs.manual == 'true' && ' --manual' || '' }}
+    - name: Create PR
+      id: cpr
+      uses: peter-evans/create-pull-request@v5
+      with:
+        commit-message: "task: vlaura vote report"
+        title: "vlaura vote report"
+        branch: "gha-biweekly-vlaura-votes-${{ github.run_id }}"
```

Comment:
I don't think you need the run-id here nor am I sure it works.  The branch-suffix below will add something to the end of the branch.  Maybe you can also use run_id.  Check the docs for the create-pull-request action

Reply:
adjusted so that the generated branches and reports use a date: c73267f